Logos/OrcaSecurity.svg

```diff
@@ -1,14 +1,16 @@
-        <path id="path5" d="" stroke="none" fill="#080404" fill-rule="evenodd"></path>
-        <path id="path6" d="" stroke="none" fill="#080404" fill-rule="evenodd"></path>
-        <path id="path7" d="" stroke="none" fill="#080404" fill-rule="evenodd"></path>
-        <path id="path8" d="" stroke="none" fill="#080404" fill-rule="evenodd"></path>
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Please remove the xlink part specifically this one - 'xmlns:xlink="http://www.w3.org/1999/xlink"'